packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/figure--image.twig

```diff
@@ -19,7 +19,7 @@
     'title.text': figure.title.text,
     'title.visible': figure.title.visible|default(false),
     'wrap': wrap,
-    'class': '',
+    'class': 'ma__image',
```

Comment:
@arthurbaghdas After speaking to Yukiko about the failed Backstop tests, we think this class is causing some spacing issues, since this class exists in `@atoms/09-media/image.twig`. Could we change this class to something else like `ma__figure-image` along with making any other CSS adjustments that are needed?

Reply:
Fixed the issues. Please review again.